### ISSUE TITLE
Is this active / maintained still?

### ISSUE THREAD

**Issue by lukehinds** [sigstore/sigstore-helm-operator#9]
What's the status of this repo, if not maintained we can place into archive?

cc @sabre1041 @dlorenc @lcarva @robnester-rh

**Comment by lcarva** [comment]
It doesn't look very active, and it has fallen behind (rekor 0.7.0). It was useful when my team had to spin up a Rekor instance. We don't do it now, but we will again at some point. I guess if we retire this repo, what's the prescribed way to deploy Rekor?

**Comment by sabre1041** [comment]
@lukehinds I do not believe that this is active/being maintained so I would recommend that we archive

@lcarva You can leverage the [helm-charts](https://github.com/sigstore/helm-charts) and in particular, the [rekor](https://github.com/sigstore/helm-charts/tree/main/charts/rekor) chart
